prosody-plugins/token/util.lib.lua

```diff
@@ -38,19 +29,12 @@ local function _verify_token(token, appId, appSecret, roomName, disableRoomNameC
 	if roomClaim == nil and disableRoomNameConstraints ~= true then
 		return nil, "'room' claim is missing";
 	end
-	if roomName ~= nil and roomName ~= roomClaim and disableRoomNameConstraints ~= true then
-		return nil, "Invalid room name('room' claim)";
-	end
 
 	return true;
 end
 
-function _M.verify_token(token, appId, appSecret, roomName, disableRoomNameConstraints)
-	return _verify_token(token, appId, appSecret, roomName, disableRoomNameConstraints);
-end
-
-function _M.get_room_name(token, appSecret)
```

Comment:
It was used - it was bound to the session and mod_token_verification.lua verifies if the user who is trying to create the room is allowed to do that.

Yes, I agree that passing it for verification at line 92 did not make much sense since we can not verify room at that point.